src/funfuzz/util/tooltool/tooltool.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 # coding=utf-8
-# pylint: disable=assignment-from-no-return,bad-python3-import,broad-except,fixme,invalid-name,logging-too-few-args
+# pylint: disable=assignment-from-no-return,broad-except,fixme,invalid-name,logging-too-few-args
```

Comment:
Rather than maintain this copy of tooltool, why not install the github version? `git+https://github.com/mozilla/build-tooltool.git`. We can also work to move this to `fuzzfetch` sooner than later if you'd rather.